.github/workflows/ci.yaml

```diff
@@ -44,6 +44,7 @@ jobs:
               - 'charts/ingress-nginx/Chart.yaml'
               - 'charts/ingress-nginx/**/*'
 
+
```

Comment:
Although it doesn't matter, we shouldn't need these new blank lines :smile:

Reply:
i did it for readability really